src/modules/CurrencyModule.h

```diff
@@ -84,27 +86,31 @@ class CurrencyModule : public Module
 
 	struct CurrencyConfig
 	{
-		double randGiveawayProb { 0.001 };
-		unsigned int freebieExpireTime { 60 };	//in seconds
-		int dailyReward { 100 };
-		int freebieReward { 300 };
-		int gambleReward { 50 };
-		int minGuess { 0 };
-		int maxGuess { 5 };
-		int gambleLoss { 10 };
-		snowflake_t giveawayChannelId { 0 };
+		double randGiveawayProb{ 0.001 };
+		unsigned int freebieExpireTime{ 60 };	//in seconds
+		int dailyReward{ 100 };
+		int freebieReward{ 300 };
+		int gambleReward{ 50 };
+		int minGuess{ 0 };
+		int maxGuess{ 5 };
+		int gambleLoss{ 10 };
+		snowflake_t giveawayChannelId{ 0 };
 		QString currencyName;
 		QString currencySymbol;
 		bool isRandomGiveawayDone{ false };
 		bool allowGiveaway{ false };
-		snowflake_t giveawayClaimer { 0 };
+		snowflake_t giveawayClaimer{ 0 };
 		QTimer* freebieTimer{ nullptr };
-		int dailyBonusAmount { 50 };
-		int dailyBonusPeriod { 3 };
-		int stealSuccessChance { 30 };
-		int stealFinePercent { 50 };
-		int stealVictimBonusPercent { 25 };
-		int stealFailedJailTime { 3 };
+		int dailyBonusAmount{ 50 };
+		int dailyBonusPeriod{ 3 };
+		int stealSuccessChance{ 30 };
+		int stealFinePercent{ 50 };
+		int stealVictimBonusPercent{ 25 };
+		int stealFailedJailTime{ 3 };
+
+		int bribeMaxAmount{ 150 };
+		int bribeLeastAmount{ 20 };
+		int bribeSuccessChance{ 69 };
```

Comment:
Please don't format code that you didn't write, so that the diff is minimal!

Reply:
Oops, that Visual Studio feature for auto format on copy and pasting. I am fixing this. Thanks! 😄

Reply:
Fixed!